deploy/examples/create-external-cluster-resources.py

```diff
@@ -25,11 +25,14 @@
 from os import linesep as LINESEP
 from os import path
 from email.utils import formatdate
-import urllib.parse
-
 import requests
 from requests.auth import AuthBase
 
+py3k = False
+if sys.version_info.major >= 3:
```

Comment:
maybe we can just keep >=3,
With a comment,
`whenever upgrading to python4 check if python4 supports it.` 

Otherwise no worries